src/test/ui/consts/issue-55878.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0080]: the type `[u8; 18446744073709551615]` is too big for the current architecture
+  --> $SRC_DIR/libcore/mem/mod.rs:LL:COL
+   |
+LL |     intrinsics::size_of::<T>()
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^ the type `[u8; 18446744073709551615]` is too big for the current architecture
```

Comment:
I'm fine merging as is with a FIXME in place for adding a useful (not duplicated) note here. It's likely a bit involeved in the const eval engine, since the way I'm imagining it requires annotating every layout_of call.

Reply:
Yeah, I thought about that too and came to the same conclusion.

Reply:
I think I have a way to make a separate label that won't be too hacky. What do you think a good wording for this error would be/what the end result should look like?

Reply:
The problem is that there are many situations in const eval that compute layouts and could thus fail. I think we should label each with their own message. If we hardcode a message for just this case it will be nonsensical in all other cases.

Reply:
I just removed the label altogether and left the main message only